test/utils.cpp

```diff
@@ -242,13 +252,24 @@ TEST_CASE("run_program()", "[utils]")
 	REQUIRE(utils::run_program(argv, "") == "hello world");
 }
 
-TEST_CASE("resolve_tilde()", "[utils]")
+TEST_CASE("resolve_tilde() replaces ~ with the $HOME directory", "[utils]")
```

Comment:
That might be too pedantic, but I'd write "with the *path* to the $HOME directory".